crates/oxc_linter/src/snapshots/prefer_todo.snap

```diff
@@ -0,0 +1,60 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 105
+expression: prefer_todo
+---
+  ⚠ eslint-plugin-jest(prefer-todo): Suggest using `test.todo`.
+   ╭─[prefer_todo.tsx:1:1]
+ 1 │ test('i need to write this test');
+   · ─────────────────────────────────
```

Comment:
The span should only cover `test` because it's where the error is.

Reply:
done